starter-core/src/main/java/io/micronaut/starter/feature/function/template/http/httpFunctionKotlinController.rocker.raw

```diff
@@ -0,0 +1,42 @@
+@import io.micronaut.starter.application.Project
+
+@args (
+Project project
+)
+
+@if (project.getPackageName() != null) {
+package @project.getPackageName()
+}
+
+
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Get
+import io.micronaut.http.annotation.Post
+import io.micronaut.http.annotation.Body
+import io.micronaut.http.HttpStatus
+
+@@Controller("/")
+class @project.getClassName()Controller {
+
+    @@Get(uri="/", produces=["text/plain"])
+    fun index(): String {
+        return "Example Response"
+    }
+
+    @@Post(produces=["application/json"])
```

Comment:
We should use the constants defined in `MediaType` instead of `text/plain` here.

Reply:
Micronaut defaults to json. 

That it is to say: 

Instead   `@Post(uri="/", produces="application/json")`

You can just write: 

`@Post`

I've done the change.